egs/librispeech/s5/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -173,6 +173,7 @@ if [ $stage -le 15 ]; then
      /export/b0{5,6,7,8}/$USER/kaldi-data/egs/librispeech-$(date +'%m_%d_%H_%M')/s5c/$dir/egs/storage $dir/egs/storage
   fi
 
+  mkdir -p $dir/egs
```

Comment:
hm.
Actually I think the better fix would be to just remove the following line `   touch $dir/egs/.nodelete `. 
that's what would fail.

Reply:
@akreal, could you please change this so it removes the line `touch $dir/egs/.nodelete` instead of adding this line?

Reply:
Sure, done.